.github/workflows/build.yaml

```diff
@@ -1,20 +1,21 @@
 name: "build"
 on: [push, pull_request]
+
 jobs:
   build:
     name: build
     runs-on: ubuntu-latest
     steps:
       - name: Setup BATS
-        uses: mig4/setup-bats@v1
-        with:
-          bats-version: 1.7.0
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y bats
 
       - name: Check out code
         uses: actions/checkout@v4
 
-      - name: Lint Dockerfile
-        uses: hadolint/hadolint-action@v3.1.0
+      - name: Run tests
+        run: bats test
```

Comment:
Can you get hadolint from their repo instead? https://github.com/hadolint/hadolint/releases/tag/v2.12.0

Reply:
No, I didn't. I believe other projects in neuvector haven't applied such actions, so we should remove it.

If we actually want to use it, should I request approval from security team first?

Reply:
I saw your change now.  Thanks for doing it!
Actually the change this time focuses on unapproved GHA, as it has a lot more privileges than one executable.  Using hadolint is fine.